uwblueprint/focus-on-nature
# Add stepper navigation in camp creation flow

## Notion ticket link

[Camp Creation/Editing: Stepper Navigation Component](https://www.notion.so/uwblueprintexecs/Camp-Creation-Editing-Stepper-Navigation-Component-001099c4e1004cc784b3e18fc8be5e09)



## Implementation description
* Created the stepper, implemented as boxes with `onClick`s
* Added dummy state in the main camp creation page component



## Steps to test
1. Go to my branch and try it out locally or see below (paying attention to when I try to go to a following page but am unable to)

https://user-images.githubusercontent.com/30396273/201606273-9bcb618d-2fbf-4eb1-8e59-7105775155aa.mp4



## What should reviewers focus on?
* Review all the design decisions/considerations I made, most of the dummy code only exists in the main component since we don't have anything like Redux right now.


## Checklist
